app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from datetime import date
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("Visitor is not vaccinated")
+        if visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError("Visitor has outdated vaccine")
+        if visitor["wearing_a_mask"] is False:
+            raise NotWearingMaskError("Visitor nis not wearing mask")
```

Comment:
There's a small typo in the error message. It should be "is not" instead of "nis not".